tests/raftstore/util.rs

```diff
@@ -121,14 +121,14 @@ pub fn new_server_config(cluster_id: u64) -> ServerConfig {
         // Considering connection selection algo is involved, maybe
         // use 2 or larger value here?
         grpc_raft_conn_num: 1,
-        end_point_concurrency: 1,
         ..ServerConfig::default()
     }
 }
 
 pub fn new_readpool_cfg() -> ReadPoolConfig {
     ReadPoolConfig {
         storage: ReadPoolInstanceConfig::default_for_test(),
+        coprocessor: ReadPoolInstanceConfig::default_for_test(),
```

Comment:
concurrency should be `1` to be same as original.

Reply:
It may hide potential mistakes due to single thread so that I intentionally set it to a larger value to run test in a concurrent environment. Although all tests are passed, I prefer keep it here.